util/dvsim/Deploy.py

```diff
@@ -456,6 +456,49 @@ def dispatch_cmd(self):
         super().dispatch_cmd()
 
 
+class CompileBuildOnly(Deploy):
+    """
+    Abstraction for building the simulation executable.
+    """
+
+    # Register all builds with the class
+    items = []
```

Comment:
Do you really want this to be a class variable, as opposed to an instance variable?

Reply:
This is present in other derivatives of `Deploy` as well. @sriyerg, is this intended?